src/frontend/Komponenter/Behandling/Brevmottakere/SøkOrganisasjon.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { Dispatch, SetStateAction, useEffect, useState } from 'react';
+import { Input } from 'nav-frontend-skjema';
+import { useApp } from '../../../App/context/AppContext';
+import { byggTomRessurs, Ressurs } from '../../../App/typer/ressurs';
+import DataViewer from '../../../Felles/DataViewer/DataViewer';
+import { Element } from 'nav-frontend-typografi';
+import { Knapp } from 'nav-frontend-knapper';
+import { IOrganisasjonMottaker } from './typer';
+import { StyledSøkInput, StyledSøkResultat } from './brevmottakereStyling';
+
+interface Props {
+    valgteMottakere: IOrganisasjonMottaker[];
+    settValgteMottakere: Dispatch<SetStateAction<IOrganisasjonMottaker[]>>;
+}
+export const SøkOrganisasjon: React.FC<Props> = ({ settValgteMottakere }) => {
+    const { axiosRequest } = useApp();
+
+    const [organisasjonsnummer, settOrganisasjonsnummer] = useState('');
+    const [navnHosOrganisasjon, settNavnHosOrganisasjon] = useState('');
+    const [organisasjonRessurs, settOrganisasjonRessurs] = useState(
+        byggTomRessurs<IOrganisasjon>()
+    );
+    const [feil, settFeil] = useState('');
+
+    useEffect(() => {
+        if (organisasjonsnummer?.length === 9) {
+            const søkOrganisasjon = () => {
+                axiosRequest<IOrganisasjon, null>({
+                    method: 'GET',
+                    url: `familie-ef-sak/api/organisasjon/${organisasjonsnummer}`,
+                }).then((response: Ressurs<IOrganisasjon>) => {
+                    settOrganisasjonRessurs(response);
+                });
+            };
```

Comment:
Gir det noe ekstra å sette denne til en const her, som man kaller på direkte etterpå? 
Burde man bare kalle på axiosRequest direkte eller flytte ut const utenfor useEffect?

Reply:
Det er en slags best practise å definere funksjoner brukt av en useEffect i selve effecten for å tydeliggjøre hvilke dependencies som skal trigge nytt kall https://reactjs.org/docs/hooks-faq.html#is-it-safe-to-omit-functions-from-the-list-of-dependencies .

Men det er ingen spesiell grunn til å ikke kalle axiosRequest direkte, så kan gjøre det 👍  (tok ikke axiosRequest med i dependency arrayet da det tidligere ville gi evig løkke, men det er visst fikset)